Lombiq.Tests.UI/Helpers/ReliabilityHelper.cs

```diff
@@ -66,7 +67,8 @@ public static class ReliabilityHelper
         {
             throw new TimeoutException(
                 "The process didn't succeed with retries before timing out " +
-                $"(timeout: {result.Wait.Timeout}, polling interval: {result.Wait.PollingInterval}).");
+                $"(timeout: {result.Wait.Timeout.ToTechnicalString()}, polling interval: " +
+                $"{result.Wait.PollingInterval.ToTechnicalString()}).");
```

Comment:
You can use the new approach here, too:
```suggestion
                $"(timeout: {result.Wait.Timeout.ToTechnicalString()}, polling interval: " +
                string.Create(
                    CultureInfo.InvariantCulture,
                    $"(timeout: {result.Wait.Timeout}, polling interval: {result.Wait.PollingInterval})."));
```